tutorial/__init__.py

```diff
@@ -2,13 +2,42 @@
 
 
 def my_sum(a, b):
+    """Add two numbers together.
+
+    Args:
+        a (numeric): The first number.
+        b (numeric): The second number.
+
+    Returns:
+        numeric: The sum of the two numbers.
+    """
     return a + b
 
 
 def my_mean(array):
+    """Calculate average of numbers
+
+    Args:
+        array (list): The array of numbers.
```

Comment:
List の中の型を以下のように記載することができます。
以下のように記載することで List の中の要素の型を読み手に伝えることができより使いやすくなると思いました。
いかがでしょうか？

```suggestion
        array (List[Union[int, float]]): The array of numbers.
```

Reply:
いいと思いましたのでこちらでアップデートしました [3ec33a2](https://github.com/yamasakih/git-tutorial/pull/9/commits/3ec33a2b6c6ac586f43054dd6a6984126203735f)